tests/e2e-permissions.spec.ts

```diff
@@ -0,0 +1,210 @@
+import { test, expect } from "./fixtures/lxd-test";
+import { visitInstance } from "./helpers/instances";
+import { gotoURL } from "./helpers/navigate";
+import { execSync } from "child_process";
+
+test.beforeEach(() => {
+  test.skip(
+    Boolean(process.env.CI),
+    "This suite is only run manually to create screenshots for the documentation",
+  );
+});
+
+test.describe("Given a user with Viewer Server permissions...", () => {
+  const instanceName = "test-instance";
+  const profileName = "test-profile";
+  const networkName = "test-network";
+  const poolName = "test-pool";
+  const volumeName = "test-volume";
+  //   const customISOName = "test-custom-iso"; // Need ISO file
+  //   const bucketName = "test-bucket"; // Need CephObject
+  const aclName = "test-acl";
+  const identityName = "test-identity";
+  const groupName = "test-group";
+  test.beforeAll(async ({}) => {
+    try {
+      execSync(`lxc init ubuntu:24.04 ${instanceName}`);
+      console.log("Instance created");
+
+      execSync(`lxc profile create ${profileName}`);
+      console.log("Profile created");
+
+      execSync(`lxc network create ${networkName}`);
+      console.log("Network created");
+
+      execSync(`lxc storage create ${poolName} dir`);
+      console.log("Storage pool created");
+
+      execSync(`lxc storage volume create ${poolName} ${volumeName}`);
+      console.log("Custom volume created");
+
+      // execSync(`lxc storage volume import ${poolName} /tmp/${customISOName}.iso ${customISOName} --type iso`);
+      // console.log("Custom ISO imported");
```

Comment:
I think you can import any random file here, we could do an exec to create a file in tmp that we import in the next exec.